src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -167,11 +171,17 @@ export class DictionaryConcepts extends Component {
 
   handleDelete = () => {
     const { data, collectionName, type } = this.state;
-    this.props
-      .removeDictionaryConcept(data, type, this.props.match.params.typeName, collectionName);
-  }
+    this.props.removeDictionaryConcept(
```

Comment:
Is this a formatting change?

Reply:
I had corrected this formatting changes on main branch.